os/arch/arm/src/s5j/Kconfig

```diff
@@ -358,3 +358,11 @@ config S5J_SENSOR_PPD42NS
 	default n
 
 endmenu
+
+comment "S5J PM Options"
+config S5J_PM
+	bool "S5J PM"
+	default n
+	select ARCH_HAVE_PM
```

Comment:
This is a different usage against another ARCH_HAVE_XX config.
Usually ARCH_HAVE_XX shows architecture supporting XX and it is selected by another config of board or chip automatically, not by user.
And XX config provides selecting that functionallity by user.

In other words, ```select ARCH_HAVE_PM``` should be at ```ARCH_BOARD_ARTIKxx```. And this config should have ```depends on ARCH_HAVE_PM```.